src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,63 @@
+package duke.task;
+
+import duke.datetime.DateTime;
+import duke.exception.DukeException;
+
+/**
+ * Events are tasks that start at a specific time and ends at a specific time.
+ */
+public class Event extends Task {
+
+    protected DateTime dateTime;
```

Comment:
Your dateTime for Event is just like Deadline.
To differentiate it from Deadline, probably you could try implementing Event with startTime and endTime? In this way Events can represent a duration like the Events in Google Calendar.

Reply:
Yep, I just realised it's a little weird that my events do not have an end time.
I'll implement that soon, thanks for the suggestion!